### DISCUSSION TITLE
Docker installation on Synology: Reuse of influxdb and redis?

### DISCUSSION THREAD

**Discussion by fugazzy** [solectrus/solectrus#3266] [General]
Hello, I am running Solectrus on my Synology DS in its Container Manager. 
I also want to use Paperless, that is also running in Container Manager and also uses redis.
And I would like to store data from my Home Assistant (running where else) to the influxdb.

Is it possible to use the already installed container (influx2, redis)? Or use their official containers with solectrus?
Is it possible to include Paperless in watchtower?

**Comment by ledermann** [comment]
I strongly advise to **not** mix SOLECTRUS containers with other applications. Why?

1) In case of any issue, it is much more difficult to track it down.
2) When multiple apps use a database container, it's much harder to do updates. You have to ensure that an update is ok for all applications.
3) If you don't have a shortage of RAM, it's not worth it. Docker is made to manage lots of containers. Just let Docker do its work ;-)

To use Watchtower for other apps, just add another Watchtower container to their compose file.

**Reply by frizzle0815** [reply]
I am also using Paperless (updated with a separate watchtower) and keep everything separated without problems.

I suggest having a look at Portainer to manage your docker containers instead of using Synologys Container Manager. There are many youtube tutorials about this.

**Comment by fugazzy** [comment]
OK, thank you! I now managed to run all separately.
Now I would like to integrate tibber-collector and senec-charger.

„It's recommended to integrate the tibber-collector into your [SOLECTRUS hosting](https://github.com/solectrus/hosting).“

I could not find the information to modify the docker-compose.yaml only the .env.

**Reply by ledermann** [reply]
You are right, the README of `tibber-collector` is a bit vague. I have just fixed that.

You can find an `compose.yml` example in the `senec-charger` repository:
https://github.com/solectrus/senec-charger/blob/develop/compose.yml
